main.js

```diff
@@ -12,4 +12,133 @@ popUp.addEventListener('click', () => {
+    img: 'img/Portfolio4-big.svg',
+    tech: ['html', 'css', 'javaScript'],
+    live: 'https://enekwestanley.github.io/Portifolio-page/',
+    source: 'https://github.com/Enekwestanley/Portifolio-page',
+  },
+};
+
+const modalSubt = document.getElementById('modalSubt');
+const modalTech = document.getElementById('modalTech');
+
+function addCounter() {
+  const counter = document.createElement('img');
+  counter.src = 'img/Counter.svg';
+  counter.alt = 'counter';
+  modalSubt.appendChild(counter);
+}
+
+function addSubt(modal) {
+  const firstDiv = document.createElement('div');
+  firstDiv.className = 'font500 black font18-24';
+  modalSubt.appendChild(firstDiv);
+  [firstDiv.innerHTML] = modalObj[modal].subt;
+  modalObj[modal].subt.slice(1).forEach((i) => {
+    addCounter();
+    const otherDiv = document.createElement('div');
+    otherDiv.className = 'gray font18-24';
+    modalSubt.appendChild(otherDiv);
+    otherDiv.innerHTML = i;
+  });
+}
+
+function addTechs(modal) {
+  modalObj[modal].tech.forEach((i) => {
+    const li = document.createElement('li');
+    li.appendChild(document.createTextNode(i));
+    modalTech.appendChild(li);
+  });
+}
+
+function clearModal() {
+  modalTech.innerHTML = '';
+  modalSubt.innerHTML = '';
+}
+
+function popModal(modal) {
+  document.getElementById('modalTitle').textContent = modalObj[modal].title;
+  addSubt(modal);
+  document.getElementById('modalDesc').textContent = modalObj[modal].desc;
+  addTechs(modal);
+  document.getElementById('modalImg').src = modalObj[modal].img;
+  document.getElementById('modalBtn1').setAttribute('onClick', `location.href='${modalObj[modal].live}';`);
+  document.getElementById('modalBtn2').setAttribute('onClick', `location.href='${modalObj[modal].source}';`);
+}
+
+const modal = document.getElementById('myModal');
+const mBtn1 = document.getElementById('myBtn1');
+const mBtn2 = document.getElementById('myBtn2');
+const mBtn3 = document.getElementById('myBtn3');
+const mBtn4 = document.getElementById('myBtn4');
+const xp = document.getElementsByClassName('xp')[0];
+
+mBtn1.onclick = function seeP1() {
+  popModal('modal1');
+  modal.style.display = 'block';
+};
+
+mBtn2.onclick = function seeP2() {
+  popModal('modal2');
+  modal.style.display = 'block';
+};
+
+mBtn3.onclick = function seeP3() {
+  popModal('modal3');
+  modal.style.display = 'block';
+};
+
+mBtn4.onclick = function seeP4() {
+  popModal('modal4');
+  modal.style.display = 'block';
+};
```

Comment:
- [ ] I see that you are manually passing in the modal that you'd like to display to your popModal method. This approach is inefficient because if you had 20 projects, then you have to this 20 times 👇 
```
mBtn4.onclick = function seeP4() {
  popModal('modal4');
  modal.style.display = 'block';
};
```
The above-highlighted code violates the DRY (Do not repeat yourself) principle.

Your modal popup should be populated with the corresponding data from a projects array in your `main.js` file This makes adding future projects to your portfolio very easy. If you have a new project to add in the future, all you have to do is to go to the projects array in your js file and add it to the array. 😎

- [ ]  I noticed that your projects section  is still hardcoded in your HTML. 
The contents of `<section class="sectiontemplate">` should be displayed dynamically.
Please refer to the projects array suggestion above to dynamically display your projects. Loop through the array and use its contents to display all your projects in the projects section. The advantage of this is that if you have a new project or you want to remove an older project, you can do it in one place only - the projects array. 🙏